src/config/webpack.dev.config.babel.js

```diff
@@ -12,24 +13,30 @@ import webpackIsomorphicToolsConfig from './webpack-isomorphic-tools';
 const webpackIsomorphicToolsPlugin =
   new WebpackIsomorphicToolsPlugin(webpackIsomorphicToolsConfig);
 const APP_NAME = config.get('currentApp');
-const BABEL_QUERY = {
-  presets: [
-    'es2015',
-    'stage-2',
-    'react',
-  ],
-  plugins: [
-    'transform-class-properties',
-    'transform-es2015-modules-commonjs',
-    ['react-transform', {
-      transforms: [{
-        transform: 'react-transform-hmr',
-        imports: ['react'],
-        locals: ['module'],
-      }],
-    }],
-  ],
-};
+
+const babelrc = fs.readFileSync('./.babelrc');
+let babelrcObject = {};
+
+try {
+  babelrcObject = JSON.parse(babelrc);
+} catch (err) {
+  console.error('ERROR: Error parsing .babelrc.');
```

Comment:
Should this just quit instead of printing an error and trying to continue on? I'm guessing it won't work without this config.